Tools/ardupilotwaf/ardupilotwaf.py

```diff
@@ -238,6 +238,7 @@ def ap_get_all_libraries(bld):
             continue
         libraries.append(name)
     libraries.extend(['AP_HAL', 'AP_HAL_Empty'])
+    libraries.append('AP_PiccoloCAN/piccolo_protocol')
```

Comment:
drive by?

Reply:
No, required.  The individual commit has a comment on it - but basically the examples instantiate their own very-special-vehicles, and those very-special-vehicles can no longer entirely elide the code magically, so we need to include this now.  Only examples are affected.